spec/tail_spec.js

```diff
@@ -1,10 +1,38 @@
-var tail = require("../src/tail.js")
+import {tail} from '../src/tail'
 
 describe('tail', ()=> {
   it('returns the first element in an array', () => {
-    expect(tail([1,2,3,4,5])).toEqual([2,3,4,5])
-    expect(tail([2,3,4,5,])).toEqual([3,4,5])
-    expect(tail([3,4,5,])).toEqual([4,5])
-    expect(tail([{first: 'first'},{second: 'second'},{third: 'third'},{fourth: 'fourth'},{fifth: 'fifth'},])).toEqual([{second: 'second'},{third: 'third'},{fourth: 'fourth'},{fifth: 'fifth'}])
+    var array = [1,2,3,4,5]
+    expect(tail(array))
+    .toEqual([2,3,4,5])
+  }),
+
+  it('returns the first element in an array', () => {
+    var array2 = [2,3,4,5]
+    expect(tail(array2))
+    .toEqual([3,4,5])
+  }),
+
+  it('returns the first element in an array', () => {
+    var array3 = [3,4,5,]
+    expect(tail(array3))
+    .toEqual([4,5])
```

Comment:
indent, or move to previous line (the latter is preferred since this is such a short line)